config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,19 @@
+if Rails.env.test?
+  CarrierWave.configure do |config|
+    config.storage = :file
+    config.enable_processing = false
+  end
+else
+  CarrierWave.configure do |config|
+    config.fog_credentials = {
+      provider: 'AWS',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.